tests/unittests/BackendTest.cpp

```diff
@@ -497,3 +497,18 @@ INSTANTIATE_TEST_CASE_P(JIT, BackendTest, ::testing::Values(BackendKind::CPU));
 INSTANTIATE_TEST_CASE_P(OpenCL, BackendTest,
                         ::testing::Values(BackendKind::OpenCL));
 #endif // GLOW_WITH_OPENCL
+
+// check if the dump function works.
+TEST(BackendTest, dumpType) {
```

Comment:
The test is good, because it checks if APIs are present at all. On top of it could also test the correctness of the output, e.g. of tyB->toString() matches the expected string literal.